### PR TITLE
(core) set rebake option based on existing bake stage

### PR DESCRIPTION
**Pull request by anotherchrisberry**
corresponding change for https://github.com/spinnaker/orca/pull/1085

### CODE REVIEW

File: app/scripts/modules/core/delivery/manualExecution/manualPipelineExecution.controller.js [hunk 1]
```diff
@@ -115,7 +115,12 @@ module.exports = angular.module('spinnaker.core.delivery.manualPipelineExecution
       addTriggers();
       this.triggerUpdated();
 
-      this.showRebakeOption = pipeline.stages.some((stage) => stage.type === 'bake');
+      let bakeStage = pipeline.stages.find(s => s.type === 'bake');
```
Comment (duftler) [inline app/scripts/modules/core/delivery/manualExecution/manualPipelineExecution.controller.js]: What is the behavior if the pipeline has more than one bake stage, and the rebake setting is not the same for all?

Comment (anotherchrisberry) [inline app/scripts/modules/core/delivery/manualExecution/manualPipelineExecution.controller.js]: In that case, we'll set the initial value to whatever the first bake stage has configured for `rebake`, but the option will apply to all bake stages in the pipeline.

Not sure if there are any pipelines in the wild that are performing multiple bakes with divergent values for the `rebake` field, but if there are, they're probably already confused by the single checkbox option we're presenting.

**Comment by anotherchrisberry** [comment]
Haven't forgotten about this PR, just haven't made it a priority. As @duftler pointed out, we really need to look at all bake stages, and if some of them are set to always rebake and some are not, and the user doesn't want to override all of them, we need to omit the `rebake` field altogether.

I'll try to get this updated this weekend.

**Comment by jrsquared** [comment]
`I'll try to get this updated this weekend.` Good try.

**Comment by anotherchrisberry** [comment]
I'm obviously not going to get this done any time soon. If someone else wants to take a crack at it, please do!

**Pull request closed without merging**